### ISSUE TITLE
Is the code going to be released soon?

### ISSUE THREAD

**Issue by zbwxp** [robot-learning-freiburg/amodal-panoptic#2]
Your work is amazing! When is it possible that the code can be released?

If the code is not able to release very soon, could you please provide the code for the metric part (APQ) first? 
Or, is it possible that you could release the predictions on the datasets made by the model (masks in polygon form like in cocoapi) so that we can make comparisons and visulize?

Thank you very much~

**Comment by HanshengGUO** [comment]
Yes, also waiting here. Maybe @mohan1914 can firstly release validation code on the codalab server so that we can do some preparation work now.
